jsx/Filter.js

```diff
@@ -37,15 +37,21 @@ class Filter extends Component {
    * @param {string} type - type of the form element
    */
   onFieldUpdate(name, value, id, type) {
+    console.warn(value);
```

Comment:
Why would this be a warning?

Reply:
Left over from debugging. I'll take it out.